### PR TITLE
feat: add gatsby-remaker-embedder

### PR DESCRIPTION
**Pull request by frontexpert**
issue #74 
@andronovhopf  This is a solution to your suggestion.

For you by letting you just copy-paste the link to the tweet you want to embed right from within your browser onto a separate line (surrounded by empty lines) and replace it with the proper embed-code.

72981310-412f0280-3dbb-11ea-929a-bb145e992849
72981301-3e341200-3dbb-11ea-860b-1fecabfdea28
72981295-3aa08b00-3dbb-11ea-9d5e-566ec3a91e09

### CODE REVIEW

**Comment by frontexpert** [comment]
@shcheklein I have replaced all the Twitter, Youtube embeds with gatsby-remaker-embedder.
Screenshot at Jan 24 14-42-14

Please check it and let me know your feedback.
Thanks.

File: content/blog/2019-12-14-november-19-dvc-heartbeat.md [hunk 1]
```diff
@@ -31,13 +31,17 @@ involved our community is and can’t wait to share more with you:
...
+  https://youtu.be/RHQXK7EC0jI
 
 - [Hacktoberfest](https://blog.dataversioncontrol.com/dvc-org-for-hacktoberfest-2019-ce5320151a0c)
   was a great exercise for DVC team on many levels and we really enjoyed
   supporting new contributors. Kudos to
   [Nabanita Dash](https://twitter.com/explorer_07) for organizing a cool
-  [DVC-themed hackathon](https://twitter.com/psociiit/status/1185150096792535040?s=20)!
+  DVC-themed hackathon!
+
+  https://twitter.com/psociiit/status/1185150096792535040?s=20
```
Comment (shcheklein) [inline content/blog/2019-12-14-november-19-dvc-heartbeat.md]: no need `s=20` at the end?

Comment (frontexpert) [reply content/blog/2019-12-14-november-19-dvc-heartbeat.md]: Ohh!
You are right. `s=20`. it is not necessary. I forgot to remove it. Sorry. I will remove it when I commit on other tasks.
Thanks.

Comment (shcheklein) [reply content/blog/2019-12-14-november-19-dvc-heartbeat.md]: np! it's a minor thingy. thanks.

**Pull request merged**